pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "switchbot_api"
-version = "1.3.0"
+version = "1.4.0"
```

Comment:
```suggestion
version = "2.0.0"
```
I think it a version as the error handling need to be updated

Reply:
Good callout!